MediaManager/Plugin.MediaManager.Abstractions/IMediaManager.cs

```diff
@@ -63,11 +63,6 @@ public interface IMediaManager : IPlaybackManager
         event MediaFileFailedEventHandler MediaFileFailed;
 
         /// <summary>
-        /// Creates new MediaFile object, adds it to the queue and starts playing
-        /// </summary>
-        Task Play(string url, MediaFileType fileType);

```

Comment:
The problem here is that you would also have to specify the ResourceAvailability which is not possible. Otherwise you would have to add another parameter with overloading and default values... All of this just becomes a code duplication of the MediaFile constructor. Why not let people construct the MediaFile themselves and the hand it to the MediaManager? To avoid all this clutter and code duplication I thought it would be easier to simply remove the method. The user can himself see that the method requires a IMediaFile and lookup the constructor for the MediaFile class.